app/helpers/application_helper.rb

```diff
@@ -121,6 +121,17 @@ def fa_icon(icon, attributes = {})
     content_tag(:i, nil, attributes.merge(class: class_names.join(' ')))
   end
 
+  def material_symbol(icon, attributes = {})
+    tag.i(
```

Comment:
Is there a reason (retro-compatibility with existing selectors?) to keep an `<i>` tag?

If not, this should be changed to a `<span>`, as `<i>` has a semantic meaning unrelated to icons.

An alternative would be to directly insert the `<svg>` tag without a wrapper, but this would require to have `inline_svg` back (and might cause perf issues if the file needs to be parsed for each render?)

Reply:
No reason to keep the `<i>`, definitely a holdover from how font awesome styles things.

Pushed an update which uses a `span` instead, adds `role=img`, and adjusts one of the previous `.fa` styles to apply to this element as well.